docs/guide/users.md

```diff
@@ -24,16 +24,10 @@ Open the users manager interface by clicking the "Users" button in the admin men
 
 ![The admin menu at the top of the browser window with the Users button highlighted](/images/users-admin-bar.png)
 
-When the manager opens, click the "New User" button to open a fresh content editor modal. Populate the required fields, including the user's "Display Name," [role](#user-roles), username, and password.
+When the manager opens, click the "New User" button to open a fresh content editor modal. Populate the required fields, including the user's "Display Name," [role](#user-roles), username, and password. At the same time, you can set the user admin UI language if the `@apostrophecms/i18n` module is configured for multiple `adminLocales` languages. See the [reference documentation for the `i18n` module](/reference/modules/i18n.html) for more information. If this hasn't been configured, then this selection input won't be shown. Otherwise, it will allow the user to select for the admin UI to match the language for the locale, or remain set to a specific language, no matter the locale language.  
```

Comment:
This is still a little confusing.

How about:

"By default, if a site is configured for multiple locales [link here], and a user is viewing the site in a particular locale, then the user interface will also be displayed in that locale whenever possible. However, if the `adminLocales` option of the `i18n` module is set to an array of locales, then it becomes possible to 'lock' the admin UI to a particular language on a per-user basis. Users can change this setting themselves if the Personal Settings module is configured."

Reply:
Also see `defaultAdminLocale` which needs to be covered here.

Reply:
Changed.